<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DualButton</class>
 <widget class="QWidget" name="DualButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>180</width>
     <height>290</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>DualButton</string>
   </property>
   <widget class="QPushButton" name="button_r">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Button R</string>
    </property>
   </widget>
   <widget class="QWidget" name="statusLED_r" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="statusLED_l" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_l">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Button L</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>manual</string>
    </property>
   </widget>
   <zorder>statusLED_r</zorder>
   <zorder>statusLED_l</zorder>
   <zorder>button_l</zorder>
   <zorder>button_r</zorder>
   <zorder>check</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
